tests/settings.py

```diff
@@ -2,7 +2,12 @@
 import os
 
 DEBUG = True
-TEMPLATE_DEBUG = DEBUG
+
+BASE_PATH = os.environ.get('BASE_PATH', os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..')))
+
+PROJECT_PATH = os.environ.get('PROJECT_PATH', os.path.join(
+    BASE_PATH))
```

Comment:
Yes, looking at this now it seems that PROJECT_PATH is completely unnecessary. I'll remove this.